src/components/Table/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { DataGrid, GridColDef, type GridValidRowModel } from "@mui/x-data-grid";
```

Comment:
try to  import like  this   import GridValidRowModel from "@mui/x-data-grid/GridValidRowModel";